packages/3id-did-resolver/src/index.ts

```diff
@@ -52,6 +52,9 @@ export function wrapDocument(content: any, did: string): DIDDocument | null {
     verificationMethod: [],
     authentication: [],
     keyAgreement: [],
+    assertionMethod: [],
```

Comment:
Did this sneak in somehow?

Reply:
was part of a hacky workaround to fix tests when 3id-resolver upstream dep changed, its removed now